### PR TITLE
Issue #9464: Added example for LITERAL_BREAK in TokenTypes.java"

### PR DESCRIPTION
**Pull request by suubh**
Issue #9464 
Screenshot (414)

**My test code** 
```
public class test{
    public static void main(String args[]){
        
        for(int i=0;i<10;i++){
            if(i == 5){
                break;
            }
            System.out.println(i);
        }
    }
}
```
**AST** 
```
C:\Users\HP\Desktop\checklist>java -jar checkstyle-8.41-all.jar -t test.java
CLASS_DEF -> CLASS_DEF [1:0]
|--MODIFIERS -> MODIFIERS [1:0]
|   `--LITERAL_PUBLIC -> public [1:0]
|--LITERAL_CLASS -> class [1:7]
|--IDENT -> test [1:13]
`--OBJBLOCK -> OBJBLOCK [1:17]
    |--LCURLY -> { [1:17]
    |--METHOD_DEF -> METHOD_DEF [2:4]
    |   |--MODIFIERS -> MODIFIERS [2:4]
    |   |   |--LITERAL_PUBLIC -> public [2:4]
    |   |   `--LITERAL_STATIC -> static [2:11]
    |   |--TYPE -> TYPE [2:18]
    |   |   `--LITERAL_VOID -> void [2:18]
    |   |--IDENT -> main [2:23]
    |   |--LPAREN -> ( [2:27]
    |   |--PARAMETERS -> PARAMETERS [2:39]
    |   |   `--PARAMETER_DEF -> PARAMETER_DEF [2:39]
    |   |       |--MODIFIERS -> MODIFIERS [2:39]
    |   |       |--TYPE -> TYPE [2:39]
    |   |       |   `--ARRAY_DECLARATOR -> [ [2:39]
    |   |       |       |--IDENT -> String [2:28]
    |   |       |       `--RBRACK -> ] [2:40]
    |   |       `--IDENT -> args [2:35]
    |   |--RPAREN -> ) [2:41]
    |   `--SLIST -> { [2:42]
    |       |--LITERAL_FOR -> for [4:8]
    |       |   |--LPAREN -> ( [4:11]
    |       |   |--FOR_INIT -> FOR_INIT [4:12]
    |       |   |   `--VARIABLE_DEF -> VARIABLE_DEF [4:12]
    |       |   |       |--MODIFIERS -> MODIFIERS [4:12]
    |       |   |       |--TYPE -> TYPE [4:12]
    |       |   |       |   `--LITERAL_INT -> int [4:12]
    |       |   |       |--IDENT -> i [4:16]
    |       |   |       `--ASSIGN -> = [4:17]
    |       |   |           `--EXPR -> EXPR [4:18]
    |       |   |               `--NUM_INT -> 0 [4:18]
    |       |   |--SEMI -> ; [4:19]
    |       |   |--FOR_CONDITION -> FOR_CONDITION [4:21]
    |       |   |   `--EXPR -> EXPR [4:21]
    |       |   |       `--LT -> < [4:21]
    |       |   |           |--IDENT -> i [4:20]
    |       |   |           `--NUM_INT -> 10 [4:22]
    |       |   |--SEMI -> ; [4:24]
    |       |   |--FOR_ITERATOR -> FOR_ITERATOR [4:26]
    |       |   |   `--ELIST -> ELIST [4:26]
    |       |   |       `--EXPR -> EXPR [4:26]
    |       |   |           `--POST_INC -> ++ [4:26]
    |       |   |               `--IDENT -> i [4:25]
    |       |   |--RPAREN -> ) [4:28]
    |       |   `--SLIST -> { [4:29]
    |       |       |--LITERAL_IF -> if [5:12]
    |       |       |   |--LPAREN -> ( [5:14]
    |       |       |   |--EXPR -> EXPR [5:17]
    |       |       |   |   `--EQUAL -> == [5:17]
    |       |       |   |       |--IDENT -> i [5:15]
    |       |       |   |       `--NUM_INT -> 5 [5:20]
    |       |       |   |--RPAREN -> ) [5:21]
    |       |       |   `--SLIST -> { [5:22]
    |       |       |       |--LITERAL_BREAK -> break [6:16]
    |       |       |       |   `--SEMI -> ; [6:21]
    |       |       |       `--RCURLY -> } [7:12]
    |       |       |--EXPR -> EXPR [8:30]
    |       |       |   `--METHOD_CALL -> ( [8:30]
    |       |       |       |--DOT -> . [8:22]
    |       |       |       |   |--DOT -> . [8:18]
    |       |       |       |   |   |--IDENT -> System [8:12]
    |       |       |       |   |   `--IDENT -> out [8:19]
    |       |       |       |   `--IDENT -> println [8:23]
    |       |       |       |--ELIST -> ELIST [8:31]
    |       |       |       |   `--EXPR -> EXPR [8:31]
    |       |       |       |       `--IDENT -> i [8:31]
    |       |       |       `--RPAREN -> ) [8:32]
    |       |       |--SEMI -> ; [8:33]
    |       |       `--RCURLY -> } [9:8]
    |       `--RCURLY -> } [10:4]
    `--RCURLY -> } [11:0]
```
**Expected update for Javadoc** 
```
 |--LITERAL_BREAK -> break [6:16]
 |     `--SEMI -> ; [6:21]
```

### CODE REVIEW

**Comment by suubh** [comment]
@pbludov Please have a look .

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 1]
```diff
@@ -1799,6 +1799,16 @@
     /**
      * The {@code break} keyword.  The first child is an optional
      * identifier and the last child is a semicolon.
+     * <p>For example:</p>
+     * <pre>
+     *      break;
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * +--LITERAL_BREAK -&gt; break
```
Comment (pbludov) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:1808]: Please remove `+`

File: src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java [hunk 2]
```diff
@@ -1799,6 +1799,16 @@
...
+     *      |
+     *      +--SEMI -&gt; ;
```
Comment (pbludov) [inline src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java:1810]: Please remove `|`. No need for empty lines here.
Please replace `+` with ` to match the tool output:
```
    |       |       |       |--LITERAL_BREAK -> break [6:16]
    |       |       |       |   `--SEMI -> ; [6:21]
```

**Review by pbludov — CHANGES_REQUESTED** [review]

**Comment by suubh** [comment]
Made the changes . @pbludov

**Comment by pbludov** [comment]
> Made the changes 

There should be only one commit in the PR. Please squash and rebase to the latest upstream.

**Comment by suubh** [comment]
> > Made the changes
> 
> There should be only one commit in the PR. Please squash and rebase to the latest upstream.

Okay

**Comment by muhlba91** [comment]
@suubh does it work locally?

also, can you please force push your change with the squashed commit. we‘ll then see if the CI builds and what the changeset is. 

(maybe some rebase error happened.)

**Comment by suubh** [comment]
> @suubh does it work locally?
> 
> also, can you please force push your change with the squashed commit. we‘ll then see if the CI builds and what the changeset is.
> 
> (maybe some rebase error happened.)

Yes it is working locally .

**Comment by CryptoSingh1337** [comment]
@suubh 
1. type `git rebase -i HEAD~4`
2. then squash the commits by replacing `pick` with `s` and save the file.
3. then remove the unwanted commit messages.

**Pull request closed without merging**